openid-caep-interoperability-profile-1_0.md

```diff
@@ -175,6 +193,28 @@ Receivers MUST be prepared to accept events with any of the subject identifier f
 ## Event Signatures
 All events MUST be signed using the `RS256` algorithm using a minimum of 2048-bit keys.
 
+## OAuth Service
+
+### Authorization Server
+* SHALL distribute discovery metadata (such as the authorization endpoint) via the metadata document as specified in [RFC8414]{{RFC8414}}
```

Comment:
SHALL and MUST are the same thing (I think), any particular reason you're using "SHALL" in the first bullet and "MUST" in the second one?